epochs: 100
batchsize: 10
stopping_patience: 50
data_augmentation: True
balanced_loss: True
normalization_type: group
learning_rate: 0.0001
